docs/content/migration.md

```diff
@@ -0,0 +1,545 @@
+| tabs               |   [Docs][64] |   [Docs][13] |                                                        |
+| textarea           |   [Docs][52] |   [Docs][5]  |                                                        |
+| toolbar            |   [Docs][66] |   [Docs][7]  |                                                        |
+| tooltip            |   [Docs][67] |   [Docs][18] |                                                        |
+| truncate           |   [Docs][103]|   -          | Not implemented                                        |
+| virtual-repeat     |   [Docs][68] |   [Docs][40] | `cdk-virtual-scroll-viewport` and `*cdkVirtualFor`     |
+| whiteframe         |   [Docs][104]|  [Guide][105]| Based on classes and mixins                            |
+
+### Services
+
+| Service          | Old          | New          | Special Notes                                          |
+|------------------|--------------|--------------|--------------------------------------------------------|
+| $mdAriaProvider  |   [Docs][124]|   -          | Not implemented                                        |
+| $mdBottomSheet   |   [Docs][42] |   [Docs][38] | `MatBottomSheet`                                       |
+| $mdColors        |   [Docs][125]|   -          | [`mat-color` mixin][74] supports static theme color lookups.|
+| $mdCompiler      |   [Docs][126]|   -          | Not implemented                                        |
+| $mdCompilerProvider| [Docs][127]|   -          | Not implemented                                        |
+| $mdDateLocaleProvider|[Docs][128]|  [Docs][129]| `MomentDateAdapter`                                    |
+| $mdDialog        |   [Docs][48] |   [Docs][22] | `MatDialog`                                            |
+| $mdGestureProvider|  [Docs][130]|   -          | Not implemented                                        |
+| $mdIcon          |   [Docs][131]|   [Docs][132]| `svgIcon`                                              |
+| $mdIconProvider  |   [Docs][133]|   [Docs][134]| `MatIconRegistry`                                      |
+| $mdInkRipple     |   [Docs][58] |   [Docs][19] |                                                        |
+| $mdInkRippleProvider|[Docs][135]|   [Docs][136]| `MAT_RIPPLE_GLOBAL_OPTIONS`                            |
+| $mdInteraction   |   [Docs][137]|   -          | Not implemented                                        |
+| $mdLiveAnnouncer |   [Docs][111]|   [Docs][112]| CDK `LiveAnnouncer`                                    |
+| $mdMedia         |   [Docs][137]|   [Docs][138]| CDK `BreakpointObserver.isMatched()` or Flex Layout's [MediaObserver service][119]|
+| $mdPanel         |   [Docs][109]|   [Docs][110]| CDK `Overlay`                                          |
+| $mdPanelProvider |   [Docs][139]|   -          | Not implemented                                        |
```

Comment:
Isn't this just the configuration API for `$mdPanel`, which is not just captured as part of `Overlay`?

Reply:
`$mdPanelProvider` allows developers to define configuration presets that will be stored within a cached presets object. It doesn't actually provide any configuration other than accepting a subset of the options for `$mdPanel.create()`. 

I wasn't able to find any idea of a "preset" or caching within the Overlay docs.